Hannah-Sten/TeXiFy-IDEA
# Chapterbib support

Fixes #881

Chapterbib allows having a bibliography per chapter. It works on \include commands.

#### Summary of additions and changes

* Support multiple bib run configs for the latex run config
* Disable DuplicateBibliographyInspection and DuplicateBibliographyStyleInspection when chapterbib is used

#### How to test this pull request

* Run main.tex, in the following example two bib run configs should be created and run

main.tex
```latex
\documentclass{article}
\usepackage[authoryear,sectionbib]{natbib}
\usepackage{chapterbib}

\begin{document}
    \include{introduction}
    \include{discrete_shape}
\end{document}
```

introduction.tex
```latex
Introduction cites~\cite{knuth90}.

\bibliographystyle{plainnat}
\bibliography{references}
```

discrete_shape.tex
```latex
Discrete shape cites~\cite{goossens93}.

\bibliographystyle{plainnat}
\bibliography{references}
```

Also please test normal automatic creation/running of bibtex/biber run configs

#### Wiki


- [x] Updated the wiki: https://github.com/Hannah-Sten/TeXiFy-IDEA/wiki/Chapterbib-support
